backend/monitoring/dashboard.py

```
@@ -746,5 +1108,520 @@ async def websocket_endpoint(websocket: WebSocket):
 
+@router.get("/social-media-status")
+async def get_social_media_status():
+    """Get social media automation status and campaign metrics (public endpoint)"""
+    try:
+        from social_media_marketing_automation import SocialMediaMarketingEngine
+        from validators.social_media_validator import SocialMediaValidator
+        
+        # Test SocialMediaMarketingEngine availability
+        social_engine_status = {"available": False, "error": None}
+        try:
+            engine = SocialMediaMarketingEngine()
```

Ruff (0.12.2)

Local variable `engine` is assigned to but never used

Remove assignment to unused variable `engine`

(F841)